### PR TITLE
Modify ASETIN and AEGLO information

### PR DESCRIPTION
**Pull request by EctobiusRexV**
Closing #72

### CODE REVIEW

File: bot/resource/cog/association/embed/asetin_embed.py [hunk 1]
```diff
@@ -13,21 +13,21 @@ class AsetinEmbed:
 
     WHO_WE_ARE_FIELD_NAME: str = "Qui sommes-nous?"
     WHO_WE_ARE_FIELD_DESCRIPTION: str = (
-        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial."
+        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial. L'ASETIN représente les intérêts des étudiants et étudiantes de premier cycle en informatique auprès des différentes instances officielles de l’Université Laval, sur les différents comités et devant les autres associations étudiantes."
```
Comment (BaboucheOne) [inline bot/resource/cog/association/embed/asetin_embed.py:16]: I think there is a mistake here. `Un lieu de rencontre convivial. L'ASETIN représente`
You need to add a \n after the full stop.
See below:
image

File: bot/resource/cog/association/embed/asetin_embed.py [hunk 2]
```diff
@@ -13,21 +13,21 @@ class AsetinEmbed:
...
     )
 
     PART_OF_FIELD_NAME: str = "Qui en fait partie?"
-    PART_OF_FIELD_VALUE: str = "IFT, GLO, IGG et certificat"
+    PART_OF_FIELD_VALUE: str = "Les étudiants en informatique, génie logiciel, informatique et gestion ainsi que les étudiants inscrits aux certificats en informatique"
```
Comment (BaboucheOne) [inline bot/resource/cog/association/embed/asetin_embed.py:20]: Maybe rework this line as a list. See below the result:
image

**Review by BaboucheOne — CHANGES_REQUESTED** [review]
Deux problèmes de reformatting

File: bot/resource/cog/association/embed/asetin_embed.py (hunk 2 above)
Comment (EctobiusRexV) [reply bot/resource/cog/association/embed/asetin_embed.py:20]: Will do

File: bot/resource/cog/association/embed/asetin_embed.py [hunk 3]
```diff
@@ -13,21 +13,21 @@ class AsetinEmbed:
 
     WHO_WE_ARE_FIELD_NAME: str = "Qui sommes-nous?"
     WHO_WE_ARE_FIELD_DESCRIPTION: str = (
-        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial."
+        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial.\n L'ASETIN représente les intérêts des étudiants et étudiantes de premier cycle en informatique auprès des différentes instances officielles de l’Université Laval, sur les différents comités et devant les autres associations étudiantes."
```
Comment (BaboucheOne) [inline bot/resource/cog/association/embed/asetin_embed.py:16]: Maybe no space between `\n` et `L'ASETIN`? See below :
With space: 
image


Without space:
image

Comment (EctobiusRexV) [reply bot/resource/cog/association/embed/asetin_embed.py:16]: Yeah I'll change that

**Review by BaboucheOne — APPROVED** [review]
Super!

**Pull request merged**